.gitmodules

```diff
@@ -24,3 +24,6 @@
 [submodule "src/asterics-smarthome-knx"]
 	path = src/asterics-smarthome-knx
 	url = https://github.com/asterics/asterics-smarthome-knx.git
+[submodule "src/asterics-camerainput-eyecontrol"]
```

Comment:
checks grad nicht mehr ganz: wo wird dieses submodule jetzt eingehängt? Ich finde in `AsTeRICS` keinen `src` Ordner...

Reply:
ah, ok just saw that this changes are just for the gh-pages branch.

Reply:
yes, it's for gh-pages because there the current Asterics webpage for developers is managed.